src/lib/openapi/spec/event-schema.ts

```diff
@@ -1,51 +1,136 @@
 import { FromSchema } from 'json-schema-to-ts';
 import { tagSchema } from './tag-schema';
+import { IEventTypes } from '../../types';
+import { variantSchema } from './variant-schema';
 
+const eventDataSchema = {
+    type: 'object',
+    additionalProperties: true,
+    properties: {
+        name: {
+            type: 'string',
+        },
+        description: {
+            type: 'string',
+        },
+        type: {
+            type: 'string',
```

Comment:
// I wonder if we can inject the enum type here so we don't have to maintain the list in two places
```suggestion
            type: 'string',
            example: 'APPLICATION_CREATED',
```

Reply:
Good shout! That should be possible

Reply:
Not really, cause it's not the event type, it could be the feature type here.

Reply:
The example for our feature-created events has `type: 'release'` here. So unfortunately, we can't inject the event type enum here.

Reply:
maybe we can have a union type for event-type + feature-type, but I think that'd be requesting too much on this PR. This is good for me :+1: